lib/views/order_confirmation.erb

```diff
@@ -0,0 +1 @@
+<p>Thank you for your order! Your food will be ready for pick up at <%=@current_order.pickup_time.strftime("%H:%M")%></p>
```

Comment:
Also, I notice that you set the `.strftime("%H:%M")` on `pickup_time` in the controller already. Why the repetition?